examples/data-objects/prosemirror/src/prosemirror.tsx

```diff
@@ -175,9 +152,8 @@ export class ProseMirrorFactory implements IFluidDataStoreFactory {
 				]),
 			),
 			existing,
+			async (_: IFluidDataStoreRuntime) => ProseMirror.load(runtime, context, existing),
```

Comment:
This is kind of interesting - any particular reason for using the runtime from the upper scope rather than the param here?  Not that it makes any particular difference here I think.